src/components/organisms/AlarmHistory.tsx

```diff
@@ -1,129 +1,151 @@
-    time: "15:22",
-  },
-  {
-    date: "01/01/22",
-    time: "14:22",
-  },
-  {
-    date: "01/01/22",
-    time: "14:22",
-  },
-  {
-    date: "01/01/22",
-    time: "14:22",
-  },
-  {
-    date: "01/01/22",
-    time: "14:22",
-  },
-  {
-    date: "01/01/22",
-    time: "14:22",
-  },
-  {
-    date: "01/01/22",
-    time: "14:22",
-  },
-  {
-    date: "01/01/22",
-    time: "14:22",
-  },
-  {
-    date: "01/01/22",
-    time: "14:22",
-  },
-  {
-    date: "01/01/22",
-    time: "14:22",
-  },
-  {
-    date: "01/01/22",
-    time: "14:22",
-  },
-  {
-    date: "01/01/22",
-    time: "14:22",
-  },
-  {
-    date: "01/01/22",
-    time: "14:22",
-  },
-  {
-    date: "01/01/22",
-    time: "14:22",
-  },
-  {
-    date: "01/01/22",
-    time: "14:22",
-  },
-  {
-    date: "01/01/22",
-    time: "14:22",
-  },
-  {
-    date: "01/01/22",
-    time: "14:22",
-  },
-  {
-    date: "01/01/22",
-    time: "14:22",
-  },
-  {
-    date: "01/01/22",
-    time: "14:22",
-  },
-  {
-    date: "01/01/22",
-    time: "14:22",
-  },
-];
+// const alarmData = [
+//   {
+//     date: "01/01/22",
+//     time: "14:30",
+//   },
+//   {
+//     date: "03/01/22",
+//     time: "16:30",
+//   },
+//   {
+//     date: "05/01/22",
+//     time: "08:01",
+//   },
+//   {
+//     date: "01/02/22",
+//     time: "19:03",
+//   },
+//   {
+//     date: "06/02/22",
+//     time: "12:21",
+//   },
+//   {
+//     date: "10/03/22",
+//     time: "14:22",
+//   },
+//   {
+//     date: "12/04/22",
+//     time: "09:10",
+//   },
+//   {
+//     date: "15/05/22",
+//     time: "11:20",
+//   },
+//   {
+//     date: "01/07/22",
+//     time: "15:22",
+//   },
+//   {
+//     date: "01/01/22",
+//     time: "14:22",
+//   },
+//   {
+//     date: "01/01/22",
+//     time: "14:22",
+//   },
+//   {
+//     date: "01/01/22",
+//     time: "14:22",
+//   },
+//   {
+//     date: "01/01/22",
+//     time: "14:22",
+//   },
+//   {
+//     date: "01/01/22",
+//     time: "14:22",
+//   },
+//   {
+//     date: "01/01/22",
+//     time: "14:22",
+//   },
+//   {
+//     date: "01/01/22",
+//     time: "14:22",
+//   },
+//   {
+//     date: "01/01/22",
+//     time: "14:22",
+//   },
+//   {
+//     date: "01/01/22",
+//     time: "14:22",
+//   },
+//   {
+//     date: "01/01/22",
+//     time: "14:22",
+//   },
+//   {
+//     date: "01/01/22",
+//     time: "14:22",
+//   },
+//   {
+//     date: "01/01/22",
+//     time: "14:22",
+//   },
+//   {
+//     date: "01/01/22",
+//     time: "14:22",
+//   },
+//   {
+//     date: "01/01/22",
+//     time: "14:22",
+//   },
+//   {
+//     date: "01/01/22",
+//     time: "14:22",
+//   },
+//   {
+//     date: "01/01/22",
+//     time: "14:22",
+//   },
+//   {
+//     date: "01/01/22",
+//     time: "14:22",
+//   },
+//   {
+//     date: "01/01/22",
+//     time: "14:22",
+//   },
+//   {
+//     date: "01/01/22",
+//     time: "14:22",
+//   },
+// ];
+
+type AlarmData = {
+  is_alarm: boolean;
```

Comment:
Is `is_alarm` used?

Reply:
As above

Reply:
See latest push - filter is applied using this